eventbridge-sqs-ecs-cdk/example-pattern.json

```diff
@@ -0,0 +1,57 @@
+{
+  "title": "Trigger ECS task from SQS using EventBridge",
+  "description": "Create an application to create ECS tasks based on SQS messages pushed through EventBridge.",
+  "language": "Python",
```

Comment:
Language is Typescript not python as the infra is defined in Typescript

Reply:
Oh, my bad. I have updated the file now.